Orange/widgets/unsupervised/owmds.py

```diff
@@ -451,7 +451,7 @@ def _clear_plot(self):
             self._legend_item = None
 
     def update_controls(self):
-        if self.data is None and getattr(self.matrix, 'axis', 1) == 0:
+        if self.data is None and getattr(self.matrix, 'axis', 1) in [0, 1]:
```

Comment:
`axis` is always either 0 or 1!
 
   .. attribute:: axis

        If axis=1 we calculate distances between rows,
        if axis=0 we calculate distances between columns.

This check is then redundant.